README.md

```diff
@@ -124,6 +130,14 @@ Whatever you select within `Out-ConsoleGridView` will be inserted on your comman
 
 Whatever was typed on the command line prior to hitting `F7` or `Shift-F7` will be used as a filter.
 
+### Example 9: Output processes to a tree view
+
+```PowerShell
+PS C:\> Get-Process | Show-ObjectTree
+```
+
+This command gets the processes running on the local computer and sends them to a tree view window.
```

Comment:
It may be worth stating ``Use right arrow where you see a `+` symbol to expand the tree``